### PR TITLE
Fix/9

### PR DESCRIPTION
**Pull request by CycoStallion**

### CODE REVIEW

**Comment by CycoStallion** [comment]
Fix for issue #9

File: app.js [hunk 1]
```diff
@@ -1,7 +1,12 @@
 const express = require('express');
+const { reset } = require('nodemon');
```
Comment (BhankasKartoy) [inline app.js:2]: Why is nodemon required here. Remove if not needed

File: app.js [hunk 2]
```diff
@@ -11,9 +16,19 @@ app.use((req, res, next) => {
 
 //Route for /product-page
 app.use('/product-page', (req, res, next) => {
-    res.send('<h1>This is the product page</h1>');
+    console.log(req.body); //Will show undefined w/o body parser
+    res.redirect("/");
+});
+
```
Comment (BhankasKartoy) [inline app.js:22]: Add comment to clearly mark this as post

File: package.json [hunk 3]
```diff
@@ -20,5 +20,8 @@
   "devDependencies": {
     "express": "^4.17.1",
```
Comment (BhankasKartoy) [inline package.json:21]: Express should not be dev dependency. Fix this

File: app.js (hunk 2 above)
Comment (CycoStallion) [reply app.js:21]: Done

File: package.json (hunk 3 above)
Comment (CycoStallion) [reply package.json]: Done

**Pull request merged**